src/lib/skills/firemaking.ts

```diff
@@ -0,0 +1,84 @@
+import { SkillsEnum, Burn } from '../types';
+import itemID from '../util/itemID';
+import { Emoji } from '../constants';
+
+const burns: Burn[] = [
```

Comment:
i think all the inputLogs shouldnt be a bank, and be just the itemID, no logs take more than 1 log to burn

Reply:
also why are they called `burns` and `Burn`? should be burnables/Burnable